src/cpp/library/cprover.h

```diff
@@ -22,8 +22,8 @@ void __CPROVER_assume(__CPROVER_bool assumption) __attribute__((__noreturn__));
 void __CPROVER_assert(__CPROVER_bool assertion, const char *description);
 void __CPROVER_precondition(__CPROVER_bool assertion, const char *description);
 
-void __CPROVER_input(const char *id, ...);
-void __CPROVER_output(const char *id, ...);
+void __CPROVER_input(const char *description, ...);
+void __CPROVER_output(const char *description, ...);
```

Comment:
A bit of a nuisance, but you'll need to add `// NOLINTNEXTLINE(build/deprecated)` before each of the above lines.

Reply:
Done.